snap/snapcraft.yaml

```diff
@@ -143,6 +143,11 @@ apps:
     command: usr/bin/pgbench
   pgbouncer:
     command: usr/sbin/pgbouncer
+  pgbouncer-server:
+    command: start-pgbouncer.sh
+    plugs:
+      - network
+      - network-bind
```

Comment:
This is not a service since we need templating for instances and application name. So service units would still be created in the charm code.